EmmanuelSage/EpicMail
# chore(user): Set up postgres database

#### What does this PR do?

- This PR sets up a database with a users table in it.

#### Description of Task to be completed?

- npm install pg and make-runnable and moment for dates
- Download and setup pgAdmin 4
- Create epicmail local database for testing
- Create database on heroku
- Setup heroku database on pgAdmin 4
- Create services folder for query method
- Create services/query.js file and setup a pool connection and query method
- Add services/schema.js file and add  create and drop users table query
- Add DATABASE environment variable to .env file
- Run create-tables script for local and heroku database and test



#### How should this be manually tested?

- Outlined steps on how someone else can successfully test 
  this chore to ensure it works as expected
  - $ git clone https://github.com/EmmanuelSage/EpicMail.git
  - $ cd EpicMail
  - $ npm install
  - $ npm run create-tables


#### Any background context you want to provide?

-  When testing this chore you would also need to setup PostgreSQL 
   on your machine.


#### What are the relevant pivotal tracker stories?

- PT Story link - [#164606536](https://www.pivotaltracker.com/story/show/164606536)


#### Screenshots
- Screenshot of Created users table being queried in pg Admin 4
![createdUsersTable](https://user-images.githubusercontent.com/40043468/54568812-f6916c80-49d8-11e9-9bd5-55bedca6c197.png)
